fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/service/JobName.java

```diff
@@ -56,7 +56,7 @@ public enum JobName {
     INCREASE_BUSINESS_DATE_BY_1_DAY("Increase Business Date by 1 day"), //
     INCREASE_COB_DATE_BY_1_DAY("Increase COB Date by 1 day"), //
     LOAN_COB("Loan COB"), //
-    LOAN_DELINQUENCY_CLASSIFICATION("Loan Delinquency Classification");
+    LOAN_DELINQUENCY_CLASSIFICATION("Loan Delinquency Classification"), SEND_ASYNCHRONOUS_EVENTS("Send Asynchronous Events");
```

Comment:
Please put `//` at the end of every enum value, otherwise the Spotless formatting will screw up the file.